Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -414,4 +481,12 @@ void ABenchmarkGymGameModeBase::SetTotalNPCs(int32 Value)
 void ABenchmarkGymGameModeBase::OnRepTotalNPCs()
 {
 	OnTotalNPCsUpdated(TotalNPCs);
+}
+
+int32 ABenchmarkGymGameModeBase::GetActorClassCount(TSubclassOf<AActor> ActorClass) const
+{
+	const UCounterComponent* CounterComponent = Cast<UCounterComponent>(GameState->GetComponentByClass(UCounterComponent::StaticClass()));
+	check(CounterComponent != nullptr)
+
+	return CounterComponent->GetActorClassCount(ActorClass);
```

Comment:
I guess this could actually be a static function somewhere so we don't have to rely on a counter component. Perhaps both this and counter component can reference it (or maybe a static on counter component that takes a world and class?)

Reply:
The CounterComponent will cache the actor class value. I use the game state counter component because I do not want to call too many GetAllActorsOfClass.